### DISCUSSION TITLE
Expo Config Plugin for realtime database

### DISCUSSION THREAD

**Discussion by FaisalBinAhmed** [invertase/react-native-firebase#5960] [Ideas] [answered]
Hi,
currently there's no config plugin available for the database module. As a result, I can't build my app with react-native-firebase/database from Expo custom dev client or EAS build. So I had to go back to using the JS SDK for react native.
It would be great if the config plugin for the realtime database is added in the next version.

**Comment by mikehardy** [comment, marked as answer]
Hi there! We will happily collaborate to merge PRs if someone proposes them, but Expo support is driven by the community. That said, you should closely examine if a config plugin is even needed for RTDB. It is my understanding that no config plugin is needed and you may simply use it.

**Reply by FaisalBinAhmed** [reply]
Hi Mike, in the end, I actually didn't need a separate config for the realtime database. The one for the main package worked fine. Thanks
